.github/workflows/schedule-stale.yml

```
@@ -0,0 +1,51 @@
+on:
+  schedule:
+    - cron: "0 */12 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: gatsbyjs/stale@gatsby-refactoring
```

One thing - we should lock this so any updates to that branch that we will be testing on https://github.com/LekoArts/stale-bot-test don't do weird thing on this repo - I created a `gatsby-repo-test` tag ( https://github.com/gatsbyjs/stale/tree/gatsby-repo-test ) pointing to latest commit in `gatsby-refactoring` branch. This is so we can update branch and test it on your test repo, and we feel confident, we can update tag separately
```suggestion
        uses: gatsbyjs/stale@gatsby-repo-test
```